tests/test_deformable_physics.py

```diff
@@ -108,7 +108,7 @@ def test_deformable_parallel(show_viewer):
             upper_bound=(0.33, 0.33, 1.0),
         ),
         fem_options=gs.options.FEMOptions(
-            damping=45.0,
```

Comment:
Let's keep it at 45. This test requires the velocity to be damped to zero, which won't happen if the damping is set to 0.